src/admin-bro.js

```diff
@@ -177,6 +181,8 @@ AdminBro.BaseRecord = BaseRecord
  */
 AdminBro.BaseProperty = BaseProperty
 
```

Comment:
please add the description to this property (as we have for others) - it will generate links in documentation